.github/actions/backup_and_restore/action.yml

```diff
@@ -0,0 +1,110 @@
+name: Backup_and_restore
```

Comment:
Could we rename it more explicitly? Something like "Copy database from PaaS to AKS" ?
So we don't confuse it with the backup workflow. This is only for he migration, we will remove it later on